src/main/java/io/aiven/kafka/auth/json/AivenAcl.java

```diff
@@ -118,13 +139,19 @@ public boolean equals(final Object o) {
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-        final AivenAcl aivenAcl = (AivenAcl) o;
+        return equals((AivenAcl) o);
+        // split for cyclomatic complexity check
```

Comment:
I think comments like this should not end up into final code

Reply:
Also rebasing on top of you PR (now main) should resolve this